product-service/handlers/getProductsList.js

```diff
@@ -0,0 +1,44 @@
+import { Client } from 'pg';
+
+const { PG_HOST, PG_PORT, PG_DATABASE, PG_USERNAME, PG_PASSWORD } = process.env;
+const dbOptions = {
+  host: PG_HOST,
+  port: PG_PORT,
+  database: PG_DATABASE,
+  user: PG_USERNAME,
+  password: PG_PASSWORD,
+  ssl: {
+    rejectUnauthorized: false
+  },
+  connectionTimeoutMillis: 5000,
+};
+
+export const getProductsList = async (event) => {
+  console.log(event);
+  const client = new Client(dbOptions);
+  await client.connect();
+
+  try {
+    const { rows: products } = await client.query('select * from products left join stocks on products.id = stocks.product_id');
```

Comment:
Your left join don't work correctly, you need `SELECT products.*, stocks.count FROM products LEFT JOIN stocks ON stocks.product_id=products.id` because you return `id` and `product_id`, it's same properties
![image](https://user-images.githubusercontent.com/48594817/115693100-46854f00-a368-11eb-8b74-2f80be7ed3d5.png)